### DISCUSSION TITLE
Cannot highlight resources，Create a Custom Controls Library

### DISCUSSION THREAD

**Discussion by AscendLiu** [AvaloniaUI/Avalonia#17482] [Q&A]
When I tried to customize the theme, I found that I couldn't highlight the resources I defined

![image](https://github.com/user-attachments/assets/73806f31-416d-4225-b7ef-422bc3ccf598)

![image](https://github.com/user-attachments/assets/944169b5-5567-4861-9801-579e8d0b26ae)

![image](https://github.com/user-attachments/assets/de9e3330-6616-4f1f-8522-79947c503b5d)

![image](https://github.com/user-attachments/assets/f58622f3-4848-4aab-adae-adbb3f46ac31)

Can someone help me see where I did something wrong

**Comment by StefanKoell** [comment]
Not sure if that's the issue but as long as you are in the same project, you can use simple paths as Source:
/Styles/_index.axaml

On another note, I always bootstrap resources in app.axaml. Maybe that's the issue here.

**Reply by AscendLiu** [reply]
This is a library for implementing custom themes, so there is no app.xml, and I don't think it has anything to do with app.xml because the official implementation works. I basically followed the official implementation and can't find the reason why it's not correct

**Reply by StefanKoell** [reply]
I see. Thought it was worth a shot ;)
